pages/news.js

```diff
@@ -1,16 +1,37 @@
 import React from 'react';
-import { ScrollView, Text, View } from 'react-native';
+import { ScrollView, Text, View, Share, TouchableOpacity } from 'react-native';
 import { globalStyles } from '../styles/global';
 
+const testShare = async (story) => {
+  try {
+    const result = await Share.share({
+      title:`Test Story Share`,
+      message:story,
+    });
+
+    if (result.action === Share.sharedAction) {
+      alert("Post Shared")
+    } else if (result.action === Share.dismissedAction) {
+      // dismissed
+      alert("Post cancelled")
+    }
+  } catch (error) {
+    alert(error.message);
+  }
+};
 /* This is the news text */
 export default function newsElement({ route, navigation, item }) {
+  console.log(route.params.title)
   return (
     <View>
       <ScrollView>
         <Text style={globalStyles.newsTitle}>{route.params.title}</Text>
         <Text style={globalStyles.newsParagraph}>{route.params.dateString} | {route.params.author}</Text>
         <Text style={globalStyles.newsParagraph} dataDetectorType="all" >{route.params.story}</Text>
       </ScrollView>
+      <TouchableOpacity style={globalStyles.shareBtn} onPress={()=>testShare(route.params.story)}>
+            <Text style={globalStyles.shareTxt}>SHARE</Text>
+          </TouchableOpacity>
```

Comment:
y=Your indenting looks kinda odd.
```suggestion
        <Text style={globalStyles.shareTxt}>SHARE</Text>
      </TouchableOpacity>
```